### DISCUSSION TITLE
https://seul1230.github.io/coding/sseul-boj-14003-lis5

### DISCUSSION THREAD

**Discussion by giscus** [seul1230/giscus#8] [Comments]
# https://seul1230.github.io/coding/sseul-boj-14003-lis5

sseul's blog

https://seul1230.github.io/coding/sseul-boj-14003-lis5

**Comment by J-PARK11** [comment]
해설이 꼼꼼하고 설명이 잘 되어있네요☺️
많은 도움이 되었습니다. 감사합니다.

**Reply by seul1230** [reply]
호홍 다행이네요😇 감사합니다 >^<
